### ISSUE TITLE
Better Usability - Import Statement

### ISSUE THREAD

**Issue by TanmoySG** [TanmoySG/configPy#6]
There's a usability hurdle in importing the  package. 

Currently we're using  

```
from configPy import configPy

// Cannot import the JSONConfigParser() method

confobj = configPy.JSONConfigParser(" ")
```

Instead we should be doing

```
from configPy import JSONConfigParser

confobj = JSONConfigParser(" ")
```

- [ ] Ensure better uability

**Comment by TanmoySG** [comment]
[Follow this article to publish the Package](https://towardsdatascience.com/deep-dive-create-and-publish-your-first-python-library-f7f618719e14)

**Comment by TanmoySG** [comment]
Issue Resolved in Commit [#cf3dd49](https://github.com/TanmoySG/configPy/commit/db3ce992c22239bbc2467ad8917e7922dc45688d)
